git-multimail/git_multimail.py

```diff
@@ -803,7 +803,12 @@ def _compute_values(self):
         dictionary."""
 
         values = self.environment.get_values()
-        fromaddr = self.environment.get_fromaddr(change=self)
+
+        # multimailhook.from is a global override, superceding any
+        # environment-specific mechanisms.
+        config = Config('multimailhook')
```

Comment:
This breaks one possible use of git-multimail: currently, the user can write a Python wrapper than does not use `git_multimail.py`'s `main` function, and instead instantiate `Config` and the environment in a custom way. For example, you can create a variant of `git_multimail.py` that reads config in `myhook` instead of `multimailhook`. But this only works only if all the code uses the same `Config` object.

I think (not checked) that you can access the object with `self.environment.config`.